packages/react/src/components/MultiSelect/FilterableMultiSelect.tsx

```diff
@@ -75,7 +261,7 @@ const FilterableMultiSelect = React.forwardRef(function FilterableMultiSelect(
   const [currentSelectedItems, setCurrentSelectedItems] = useState(
     initialSelectedItems ?? []
   );
-  const textInput = useRef();
+  const textInput = useRef<HTMLInputElement>();
```

Comment:
Consumers are otherwise oblivious of the `current` element type and shout about it